include/deal.II/lac/block_vector_base.h

```diff
@@ -975,6 +964,15 @@ class BlockVectorBase : public Subscriptor
   void update_ghost_values () const;
 
   /**
+   * A variable that indicates whether this vector supports distributed data
+   * storage.
+   *
+   * For the current class, the variable equals the value declared for the
+   * type of the individual blocks.
+   */
+  bool supports_distributed_data() const;
```

Comment:
The problem is that if we do that ```VectorSpaceVector``` cannot have this variable define because the variable will be true for some derived classes and false for others. ```VectorSpaceVector``` is an abstract class so I guess it could work but we would need to downcast every time we would need to know the variable.